src/sif/autocost.cc

```diff
@@ -72,6 +72,8 @@ constexpr float kMaxFactor = 100000.0f;
 // Default auto attributes
 constexpr float kDefaultAutoHeight = 1.6f; // Meters (62.9921 inches)
 constexpr float kDefaultAutoWidth = 1.9f;  // Meters (74.8031 inches)
+constexpr float kDefaultAutoLength = 5.3f; // Meters (208,661 inches)
+constexpr float kDefaultAutoWeight = 3.0f; // Metric Tons (6613,87 lbs)
```

Comment:
The default weight seems a little high maybe? I don't know too much about cars, but this looks more like the weight of a SUV. I think we'd be better off by setting it to an optimistically low value in order not to have routes failing that previously wouldn't have failed.